### PR TITLE
Modified Global Config

### PR DESCRIPTION
**Pull request by shubhamdixit863**
Fixes  https://github.com/numaproj/numaplane/issues/24

### Modifications

Removed Old values From Global Config ,Fixed unit tests.

### Verification

Verified By Running Unit tests

### CODE REVIEW

File: tests/config/testconfig.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 clusterName: "staging-usw2-k8s"
 syncTimeIntervalMs: 30000
+logLevel: 1
 cascadeDeletion: false
```
Comment (juliev0) [inline tests/config/testconfig.yaml:4]: don't we need to purge the fields from this that we're no longer using?

Comment (juliev0) [reply tests/config/testconfig.yaml:4]: also please update file below, plus [this](https://github.com/numaproj/numaplane/blob/main/config/manager/controller-config.yaml#L7) file

Comment (xdevxy) [reply tests/config/testconfig.yaml:4]: +1

File: internal/controller/config/config.go [hunk 2]
```diff
@@ -36,14 +36,8 @@ func GetConfigManagerInstance() *ConfigManager {
...
-	ClusterName             string `json:"clusterName" mapstructure:"clusterName"`
-	SyncTimeIntervalMs      int    `json:"syncTimeIntervalMs" mapstructure:"syncTimeIntervalMs"`
-	AutomatedSyncDisabled   bool   `json:"automatedSyncDisabled" mapstructure:"automatedSyncDisabled"`
-	CascadeDeletion         bool   `json:"cascadeDeletion" mapstructure:"cascadeDeletion"`
-	AutoHealDisabled        bool   `json:"autoHealDisabled" mapstructure:"autoHealDisabled"`
-	IncludedResources       string `json:"includedResources" mapstructure:"includedResources"`
-	LogLevel                int    `json:"logLevel" mapstructure:"logLevel"`
-	PersistentRepoClonePath string `json:"persistentRepoClonePath" mapstructure:"persistentRepoClonePath"`
+	LogLevel          int    `json:"logLevel" mapstructure:"logLevel"`
+	IncludedResources string `json:"includedResources" mapstructure:"includedResources"`
```
[the remark this answers is not in the document]
Comment (shubhamdixit863) [reply internal/controller/config/config.go:40]: Yes here-  https://github.com/numaproj/numaplane/blob/main/internal/sync/cache.go#L461

File: config/manager/controller-config.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: numaplane-controller-config
 data:
   config.yaml: |
-    syncTimeIntervalMs: 30000
+    logLevel: 1
     includedResources: "group=apps,kind=Deployment;\
     group=,kind=ConfigMap;group=,kind=Secret;group=,kind=ServiceAccount;\
     group=numaflow.numaproj.io,kind=;\
```
Comment (juliev0) [inline config/manager/controller-config.yaml:10]: @xdevxy I guess we shouldn't need `numaflow.numaproj.io` and `numaplane.numaproj.io` in here anymore, right?

Comment (xdevxy) [reply config/manager/controller-config.yaml:10]: yeah, we can take out that. @shubhamdixit863 can you take that out and try if you still able to deploy numaflow controller through `NumaflowControllerRollout`

File: config/samples/numaplane.numaproj.io_v1alpha1_numaflowcontrollerrollout.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: numaplane.numaproj.io/v1alpha1
 kind: NumaflowControllerRollout
 metadata:
   name: my-numaflow-controller
-  namespace: example-namespace
+  namespace: numaplane-system
```
Comment (juliev0) [inline config/samples/numaplane.numaproj.io_v1alpha1_numaflowcontrollerrollout.yaml:5]: Why was this changed?

File: config/manager/controller-config.yaml [hunk 5]
```diff
@@ -4,9 +4,7 @@ metadata:
   name: numaplane-controller-config
 data:
   config.yaml: |
-    syncTimeIntervalMs: 30000
+    logLevel: 1
```
Comment (juliev0) [inline config/manager/controller-config.yaml:7]: @afugazzotto I believe level 3 would be INFO, wouldn't it?

File: config/samples/numaplane.numaproj.io_v1alpha1_numaflowcontrollerrollout.yaml (hunk 4 above)
Comment (shubhamdixit863) [reply config/samples/numaplane.numaproj.io_v1alpha1_numaflowcontrollerrollout.yaml]: that was left by mistake ,fixed it

File: config/manager/controller-config.yaml (hunk 5 above)
Comment (xdevxy) [reply config/manager/controller-config.yaml:7]: is 1 debug?

Comment (juliev0) [reply config/manager/controller-config.yaml:7]: no, 4 is debug

Comment (juliev0) [reply config/manager/controller-config.yaml:7]: We had an issue in the previous repo to be able to use names instead of numbers. I am going to add that here too.

Comment (xdevxy) [reply config/manager/controller-config.yaml:7]: yeah, we should default to info then.

Comment (shubhamdixit863) [reply config/manager/controller-config.yaml]: @xdevxy i guess this PR is good to merge .Please have a look

**Comment by xdevxy** [comment]
@shubhamdixit863 you verified latest change in a local cluster, right?

**Review by xdevxy — APPROVED** [review]
